### DISCUSSION TITLE
Trivy is not installing latest version of a plugin

### DISCUSSION THREAD

**Discussion by fatihtokus** [aquasecurity/trivy#4130] [Q&A] [answered]
Hi team,

I have a trivy plugin([scan2html](https://github.com/fatihtokus/scan2html)) which creates an interactive html report for trivy output.
I installed it via `trivy plugin install github.com/fatihtokus/scan2html` which was fine. Now I have released a new version. But I cannot upgrade my local version to the latest. I tried to uninstall and install it, but no luck still installing the initial version! Is this a known bug of trivy?

Many thanks

**Comment by knqyf263** [comment]
The Trivy plugin is really simple now and doesn't support updating. In short, the behavior is expected. We welcome any contribution.

**Comment by fatihtokus** [comment, marked as answer]
@knqyf263 , I have re-tested this feature and looks like the mechanism works. But there is something wrong with my scan2html plugin which I have fixed. All looks good now :)
